tensorflow_addons/image/median_filter_2D_test.py

```diff
@@ -0,0 +1,102 @@
+from tensorflow.python.platform import test
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import dtypes
+from tensorflow.python.ops import array_ops
+import median_filter_2D as md
+
+import numpy as np
+
+
+class Median_2d_test(test.TestCase):
```

Comment:
Class name in CamelCase to be consistent with others : e.g. Median2DTest

Reply:
solved